.github/workflows/integration-tests.yml

```diff
@@ -93,16 +93,13 @@ jobs:
       - run: git clone https://github.com/bjlittle/geovista.git --single-branch
       - name: Install PyVista
         run: pip install -ve . # pyvista
-      - run: sudo apt-get install libgeos-dev
       - name: Install GeoVista
-        run: pip install -ve . # geovista
-        working-directory: geovista
-      - run: pip install -r requirements/pypi-optional-test.txt
+        run: pip install -ve .[test] # geovista
         working-directory: geovista
       - name: Download cartopy assets
         run: |
           wget --quiet ${CARTOPY_FEATURE}
           mkdir -p ${CARTOPY_SHARE_DIR}
           python cartopy_feature_download.py physical --output ${CARTOPY_SHARE_DIR} --no-warn
-      - run: pytest -W error
+      - run: pytest -W error -W "ignore:numpy.ndarray size changed:RuntimeWarning"
```

Comment:
@pyvista/developers Is this an acceptable compromise ?

Reply:
I'll investigate whether I require to recreate all of the `netCDF4` assets of `geovista` using the latest version of `numpy`. I'm not sure what would raise the ABI warning otherwise ...

Reply:
I'm good with this.